tests/components/material.test.js

```diff
@@ -68,6 +68,15 @@ suite('material', function () {
       assert.notOk(el.sceneEl.materials[oldMaterial.uuid]);
       assert.equal(el.sceneEl.materials[newMaterial.uuid], newMaterial);
     });
+
+    test('emits event when loading texture', function (done) {
+      var el = this.el;
+      el.setAttribute('material', 'src: url(http://localhost:9876/base/examples/_images/mozvr.png)');
+      el.addEventListener('material-texture-loaded', function (evt) {
+        assert(evt.detail.src === 'http://localhost:9876/base/examples/_images/mozvr.png', 'src in event detail is the same as the url set to be the image texture src.');
```

Comment:
Can store the URL in a var and use assert.equal, and don't need the failure message

```assert.equal(evt.detail.src, imageUrl)```